sqlalchemy-stubs/sql/functions.pyi

```diff
@@ -59,7 +59,7 @@ class FunctionElement(  # type: ignore[misc]
     ) -> ScalarFunctionColumn[_TE]: ...
     def table_valued(self, *expr: Any, **kw: Any) -> TableValuedAlias: ...  # type: ignore[override]
     def column_valued(
-        self, name: Optional[Any] = ...
+        self, name: Optional[Any], joins_implicitly: bool = False
```

Comment:
The `= ...` default should not have been deleted. This was needed to accept calls without a `name`, i.e. `.column_valued()`.